packages/preview-middleware-client/test/unit/cpe/init.test.ts

```diff
@@ -69,7 +69,7 @@ describe('main', () => {
         attachSelectionChange,
         getSelection: jest.fn().mockReturnValue([{ setSelected: jest.fn() }, { setSelected: jest.fn() }]),
         attachUndoRedoStackModified: jest.fn(),
-        getFlexSettings: jest.fn().mockReturnValue({ layer: 'VENDOR' }),
+        getFlexSettings: jest.fn().mockReturnValue({ layer: 'VENDOR', scenario: 'UiAdaptation' }),
```

Comment:
Could you please use the `scenario` const from `@sap-ux-private/control-property-editor-common`?

Reply:
Changed in: https://github.com/SAP/open-ux-tools/pull/1422/commits/83134d16a0fff1526e4dddf61c067912f7a5bdf2